sim/src/analytics.rs

```diff
@@ -252,6 +279,17 @@ impl Analytics {
     // TODO If these ever need to be speeded up, just cache the histogram and index in the events
     // list.
 
+    // Returns a list of turns(Intersections), with a delay over the threshold
```

Comment:
I don't think getter methods have much value here; the caller in `game` can just reach in and grab this data.

Reply:
That's fair enough. They aren't really doing much. Changed